tests/fast-integration/test/commerce-mock.js

```diff
@@ -165,6 +163,20 @@ describe("Commerce Mock tests", function () {
     })
   });
 
+  it("service binding usage for function should be ready", async function () {
+    const path = '/apis/servicecatalog.kyma-project.io/v1alpha1/namespaces/default/servicebindingusages';
+    await waitForK8sObject(watch, path, {}, (type, apiObj, watchObj) => {
+      if (watchObj.object.metadata.name == 'commerce-lastorder-sbu' && watchObj.object.status.conditions) {
+        return watchObj.object.status.conditions.some((c) => (c.type == 'Ready' && c.status == 'True'))
+      }
+      return false;
+    }, 90 * 1000, "Waiting for ServiceBindingUsage to be ready timeout");
+  });
```

Comment:
Please add a new line for better readability

Reply:
Done.